arc_utils/produce_notification.py

```diff
@@ -0,0 +1,23 @@
+# producer.py
+
+import json
+
+from kafka import KafkaProducer
+
+
+class Notification:
+    def __init__(self):
+        self.topic = "arc-rte-notifications"
+        broker_url = ["arc-kafka.pt-staging.com:9092"]
+        self.__producer = KafkaProducer(
+            bootstrap_servers=broker_url,
+            value_serializer=lambda v: json.dumps(v).encode("utf-8"),
+        )
+
+    def send_notification(self, notification_type, payload):
+        if notification_type == "LOGIN_SUCCESSFUL": 
```

Comment:
1. Avoid using if checks for checking. Try using dictionary-based switch cases in python to evaluate payloads
2. Use a schema validation library to evaluate if the payload is correct